test_runner/src/main/kotlin/ftl/config/android/AndroidGcloudConfig.kt

```diff
@@ -21,170 +22,171 @@ data class AndroidGcloudConfig @JsonIgnore constructor(
 ) : Config {
 
     @set:CommandLine.Option(
-        names = ["--app"],
-        description = ["The path to the application binary file. " +
-                "The path may be in the local filesystem or in Google Cloud Storage using gs:// notation."]
+            names = ["--app"],
```

Comment:
As I good remember we shoud use indentation 4 instead of 8.

Reply:
Right, after the update IntelliJ reset my settings. I reformat all configs, thanks!